package.json

```diff
@@ -20,16 +20,21 @@
     "test": "grunt test",
     "build:dev": "grunt build",
     "build:prod": "grunt build:minified",
+    "build:webpack-dev": "webpack --config webpack.dev.config.js",
+    "build:webpack-prod": "webpack --config webpack.prod.config.js",
+    "build:webpack-no-jquery": "webpack --config webpack.no-jquery.config.js",
+    "build:test": "webpack --config webpack.test.config.js",
     "prestart": "npm run build:dev",
     "start": "grunt start-server-open",
     "package": "grunt package",
-    "shrinkwrap": "grunt shrinkwrap-remove-resolved",
+    "shrinkwrap": "grunt shrinkwrap-remove-resolved-and-from",
     "ci-update-package": "ci-update-package",
     "ci-pkginfo:dataload": "ci-pkginfo -t dataload",
     "installsw": "install-with-shrinkwrap"
   },
   "devDependencies": {
     "app-root-path": "1.0.0",
+    "backbone": "1.2.1",
```

Comment:
Any libraries that we're exporting in the widget should be a dependency, not devDependency